src/components/input/DurationInput.js

```diff
@@ -34,8 +53,8 @@ const DurationInput = ({ value, onChange, onKeyDown }) => {
       </Box>
       <NumericInput
         placeholder="00"
-        min={0}
-        value={value?.minutes || ''}
+        {...(!value?.hours ? { min: 0 } : {})}
```

Comment:
What is this for? 🤔

Reply:
This makes sure that you can't go below 0 when the hour counter is 0. (If you have `1 hour`, you can decrement the minute picker and it will subtract the hour and set the minutes to `59`)